doc/api/async_context.md

```diff
@@ -147,6 +147,50 @@ this time will print the stack trace and exit. See
 Creating an async resource within the `onPropagate` callback will result in
 a recursive call to `onPropagate`.
 
+### Static method: `AsyncLocalStorage.bind(fn [, thisArg])`
```

Comment:
@flakey5 ... Let's remove the thisArg argument entirely on this method. Here and in the impl.